packages/perseus-editor/src/components/locked-line-settings.tsx

```diff
@@ -36,7 +36,7 @@ const LockedLineSettings = (props: Props) => {
     const {
         kind,
         points,
-        color: lineColor = "blue",
+        color: lineColor,
```

Comment:
Not really related to your change, but where is `lineColor` coming from? I'm like super confused. VS Code understands what it is, but I don't see where it's declared.

EDIT: nm I'm dumb. You're renaming `color` to `lineColor`. I guess that makes sense, but since it's so close to how TS assigns a type I wonder if it'd be better to be more verbose in how you're renaming destructured properties.

Reply:
What would you suggest?

Maybe something like
```
// Renaming because `color` can be too ambiguous.
const lineColor = props.color;
```
?